dev-python/argon2-cffi-bindings/argon2-cffi-bindings-21.2.0.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} pypy3 )
+inherit distutils-r1
+
+DESCRIPTION="Low-level CFFI bindings for the Argon2 password hashing library"
+HOMEPAGE="https://github.com/hynek/argon2-cffi-bindings"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="cpu_flags_x86_sse2"
+
+DEPEND="
+	app-crypt/argon2:=
+	virtual/python-cffi[${PYTHON_USEDEP}]
+"
+RDEPEND="${DEPEND}"
+
+DOCS=( CHANGELOG.md README.md )
+
+distutils_enable_tests pytest
+
+src_prepare() {
+	export ARGON2_CFFI_USE_SYSTEM=1
+	# We cannot call usex in global scope, so we invoke it in src_prepare
+	export ARGON2_CFFI_USE_SSE2=$(usex cpu_flags_x86_sse2 1 0)
+	default
```

Comment:
I'm pretty sure the eclass will tell you that you didn't call `distutils-r1_src_prepare`. It's easier to do these exports in `src_configure()` since it doesn't do anything by default.

Reply:
You're right, it makes more sense to make these calls in `src_configure`. I moved them over, and I included the call to `distutils-r1_src_configure` in case anything changes in the eclass.